packages/components/src/globals/scss/_tooltip.scss

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/scss/layer';
+
+// Tooltip Icon
+// Icon CSS only tooltip
+@mixin tooltip--icon {
```

Comment:
Great to have a SassDoc comment here, as well as to the rest of the mixins in this file.